lib/pbench/test/unit/server/test_user_management_cli.py

```diff
@@ -0,0 +1,257 @@
+                    TestUserManagement.EMAIL_SWITCH,
+                    TestUserManagement.EMAIL_TEXT,
+                    TestUserManagement.FIRST_NAME_SWITCH,
+                    TestUserManagement.FIRST_NAME_TEXT,
+                    TestUserManagement.LAST_NAME_SWITCH,
+                    TestUserManagement.LAST_NAME_TEXT,
+                    TestUserManagement.ROLE_SWITCH,
+                    "ADMIN",
+                ],
+            )
+            assert result.exit_code == 0
+            assert result.stdout == "Admin user test_user registered\n"
+            assert not result.stderr_bytes
+
+    @staticmethod
+    @responses.activate
+    def test_user_creation_with_invalid_role(client, server_config, pytestconfig):
+        with client:
+            runner = CliRunner(mix_stderr=False)
+            result = runner.invoke(
+                user_create,
+                args=[
+                    TestUserManagement.USER_SWITCH,
+                    TestUserManagement.USER_TEXT,
+                    TestUserManagement.PSWD_SWITCH,
+                    TestUserManagement.PSWD_TEXT,
+                    TestUserManagement.EMAIL_SWITCH,
+                    TestUserManagement.EMAIL_TEXT,
+                    TestUserManagement.FIRST_NAME_SWITCH,
+                    TestUserManagement.FIRST_NAME_TEXT,
+                    TestUserManagement.LAST_NAME_SWITCH,
+                    TestUserManagement.LAST_NAME_TEXT,
+                    TestUserManagement.ROLE_SWITCH,
+                    "ADMN",
+                ],
+            )
+            assert result.exit_code == 2
+
+    @staticmethod
+    @responses.activate
+    def test_valid_user_delete(client, server_config, pytestconfig):
+        with client:
+            runner = CliRunner(mix_stderr=False)
+            TestUserManagement.test_args(server_config, pytestconfig)
+
+            result = runner.invoke(user_delete, args=[TestUserManagement.USER_TEXT])
+            assert result.exit_code == 0
+            assert result.stdout == "User test_user deleted\n"
+            assert not result.stderr_bytes
+
+    @staticmethod
+    @responses.activate
+    def test_invalid_user_delete(client, server_config, pytestconfig):
+        with client:
+            runner = CliRunner(mix_stderr=False)
+            # Give username that doesn't exists to delete
+            result = runner.invoke(user_delete, args=["wrong_username"])
+            assert result.exit_code == 1
+            assert result.stdout == "User wrong_username does not exists\n"
+            assert not result.stderr_bytes
+
+    @staticmethod
+    @responses.activate
+    def test_valid_user_list(client, server_config, pytestconfig):
+        with client:
+            runner = CliRunner(mix_stderr=False)
+            TestUserManagement.test_args(server_config, pytestconfig)
+
+            result = runner.invoke(user_list,)
+            row_format = "{0:15}\t{1:15}\t{2:15}\t{3:15}\t{4:20}"
```

Comment:
I recommend that you import the format from the source code rather than trying to replicate it here.

Reply:
You might want to consider doing something similar with the header row.